### DISCUSSION TITLE
checkbox for select all folders in current path

### DISCUSSION THREAD

**Discussion by gogi100** [rejetto/hfs#204] [General]
i have one more requirement. can you set the checkbox for adding all folders in current shared path in admin-panel/shared files. the picture is

![select-all](https://user-images.githubusercontent.com/21057856/230652949-c8d824cc-aabb-4c50-aad0-286a831c41b7.jpg)

**Comment by rejetto** [comment]
if you want to add all content of folder X to be visible in folder Y, then you go to Y and set the source to be X

**Comment by gogi100** [comment]
Thanks
